Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -189,15 +168,10 @@ public struct OfflineMapAreasView: View {
         .frame(maxWidth: .infinity)
     }
     
-    /// Loads the appropriate preplanned map models depending on the updated device network connection.
+    /// Loads the online and offline preplanned map models.
     private func loadPreplannedMapModels() async {
-        isConnected = !offlineBannerIsPresented
-        
-        if isConnected {
-            await mapViewModel.loadPreplannedMapModels()
-        } else {
-            await mapViewModel.loadOfflinePreplannedMapModels()
-        }
+        await mapViewModel.loadPreplannedMapModels()
+        await mapViewModel.loadOfflinePreplannedMapModels()
```

Comment:
I'm still not convinced that we should load both online areas and local areas at the same time for each load. Do you think it makes sense to check if the online models are loaded, before loading the offline models? This way, we avoid the overhead of loading offline models that we don't need when the device is connected.

Reply:
The offline map models are now loaded conditionally.